src/main/java/Task.java

```diff
@@ -0,0 +1,64 @@
+import java.time.LocalDate;
+
+/**
+ * Class to represent a Task
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String time = "-1";
+    protected LocalDate processedTime = LocalDate.MIN;
```

Comment:
Can move this 2 variables to your event and deadline class, since not all classes that is inheriting from Task uses this variables (Todo class)